filters/NeighborClassifierFilter.cpp

```diff
@@ -134,7 +140,7 @@ void NeighborClassifierFilter::doOneNoDomain(PointRef &point, PointRef &temp,
         { return p1.second < p2.second; });
 
     // update point
-    auto oldclass = point.getFieldAs<double>(m_dim);
+    auto oldclass = point.getFieldAs<int>(m_dimId);
```

Comment:
should we be changing the type here? I guess people could use doubles as classifiers. We wouldn't care, but we're going to trim them off here by doing this.

Reply:
This matches the voting scheme of other parts of the filter. I made a note in the documentation. I don't know why you'd want to do this.